projects/storefrontapp/src/app/spartacus/spartacus-features.module.ts

```diff
@@ -131,6 +132,7 @@ if (environment.epdVisualization) {
     FooterNavigationModule,
     PageTitleModule,
     BreadcrumbModule,
+    ScrollToTopModule,
```

Comment:
I think this needs to be added to the inistallation schematics as well.

Reply:
Added the feature module to configuration. Need to validate change